test/com/facebook/buck/android/AndroidBinaryTest.java

```diff
@@ -149,12 +149,16 @@ public void testAndroidBinaryNoDx() throws Exception {
                     "gen/java/src/com/facebook/base/lib__libraryOne__output/" +
                     "libraryOne-obfuscated.jar")),
         ImmutableSet.of(
-            GEN_PATH.resolve("java/src/com/facebook/base/lib__libraryTwo__output/libraryTwo.jar")),
+            libraryTwo.getBuildTarget().getUnflavoredBuildTarget().getCellPath().resolve(
+                GEN_PATH.resolve(
+                    "java/src/com/facebook/base/lib__libraryTwo__output/libraryTwo.jar"))),
         GEN_PATH.resolve("java/src/com/facebook/base/__apk#aapt_package__proguard__/.proguard"),
         buildableContext,
         expectedSteps);
 
-    assertEquals(expectedSteps.build(), commands.build());
+    ImmutableList<Step> left = expectedSteps.build();
+    ImmutableList<Step> right = commands.build();
+    assertEquals(left, right);
```

Comment:
this seems like a spurious change.  Can you please revert it back to how it was please?